src/Microsoft.AspNetCore.SignalR.Core/IHubClients`T.cs

```diff
@@ -17,6 +17,8 @@ public interface IHubClients<T>
 
         T Group(string groupName);
 
+        T MultipleGroups(IReadOnlyList<string> groupNames);
```

Comment:
This should be called Groups

Reply:
Not a huge fan of `Groups` either because it's only one character off, but I think it's more appropriate than `MultipleGroups`.

Reply:
I put multiple groups because the HubContext already refers to the `IGroupManager` as groups. Should we just rename that instead and use Groups here?

Reply:
Hrm, why does `HubContext` implement `IHubClients`? SignalR 2.x didn't do that.

Reply:
Maybe that's the fix here. HubContext shouldn't need to implement `IHubClients`, the clients can be accessed via `.Clients`.

Reply:
That does mean we'd need to introduce a type to actually implement `Clients` (since HubContext just returns itself for that property right now)

Reply:
Okay, we can do that.

Reply:
Hmm, thinking that I might put this on hold to do that change and then renames

Reply:
We should call it Groups